apps/sertra/components/LaunchingModManPopup.tsx

```diff
@@ -0,0 +1,36 @@
+import { GetModMan } from "./GetModMan";
+import styles from "./LaunchingModManPopup.module.css";
+
+interface JoinServerProps {
+  setClose: () => void;
+}
+
+export const LaunchingModManPopup: React.FC<JoinServerProps> = ({
+  setClose,
+}) => {
+  return (
+    <div className={styles.background}>
+      <section className={styles.popUp}>
+        <section className={styles.header}>
+          <h2>Launching Mod Manager...</h2>
+          <button className={styles.closeWindow} onClick={setClose}>
+            X
```

Comment:
This should probably be an icon instead of a character?

Reply:
Yes true, I'll figure out how it should be done.